incubator/java-openliberty/templates/kafka/src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -1,10 +1,8 @@
-mp.messaging.connector.liberty-kafka.bootstrap.servers=kafka:9092
+mp.messaging.connector.liberty-kafka.bootstrap.servers=localhost:9093
 
 ## incomingTopic1 topic
 mp.messaging.incoming.incomingTopic1.connector=liberty-kafka
 mp.messaging.incoming.incomingTopic1.topic=incomingTopic1
-mp.messaging.incoming.incomingTopic1.value.serializer=org.apache.kafka.common.serialization.StringSerializer
```

Comment:
This change seems unrelated?

Reply:
I noticed these properties were unnecessary and were causing warnings - you don't need a serializer for an incoming topic and the deserializer will happily default.

Reply:
OK, probably they should be in a separate PR as not related to fixing `appsody stack validate`.

Reply:
Given the somewhat heavyweight release process,  it's okay with me to combine changes like this as long as you don't mind

Reply:
OK OK :) Just a point of principle and git hygiene.